homeassistant/components/stream/__init__.py

```diff
@@ -464,15 +464,17 @@ async def async_record(
         recorder.video_path = video_path
 
         await self.start()
-        self._logger.debug("Started a stream recording of %s seconds", duration)
 
         # Take advantage of lookback
         hls: HlsStreamOutput = cast(HlsStreamOutput, self.outputs().get(HLS_PROVIDER))
         if lookback > 0 and hls:
             num_segments = min(int(lookback // hls.target_duration), MAX_SEGMENTS)
             # Wait for latest segment, then add the lookback
             await hls.recv()
-            recorder.prepend(list(hls.get_segments())[-num_segments:])
+            recorder.prepend(list(hls.get_segments())[-num_segments - 1 : -1])
```

Comment:
This should actually have been changed when we moved to LL-HLS as the last segment is still in progress.

Reply:
Can you send this bug fix a separate PR? Would rather review in isolation than as part of larger change.

Reply:
Yeah sure. I'll take it out and submit it separately.